config/initializers/stripe.rb

```diff
@@ -1 +1,6 @@
-StripeWrapper::Charge.set_api_key(ENV['STRIPE_SECRET_KEY'])
+StripeWrapper.set_api_key(ENV['STRIPE_SECRET_KEY'])
+
+StripeEvent.configure do |events|
+  events.subscribe 'charge.succeeded', StripeWrapper::EventHandler::ChargeSucceeded.new
```

Comment:
It feels a little bit wrong to create a new class for a new state. Succeeded and Failed.